nautobot/ipam/tests/test_filters.py

```diff
@@ -1013,7 +1013,6 @@ def setUpTestData(cls):
             VLAN.objects.create(
                 vid=101,
                 name="VLAN 101",
-                location=cls.locations[0],
```

Comment:
This should still work with the way you've implemented `VLAN.__init__` and `save` right?

Reply:
Yup